docs/source/tutorials_source/platform/tutorial_aquarium_custom_metadata.py

```diff
@@ -0,0 +1,212 @@
+        images = coco_annotations['images']
+
+    # create a mapping from category id to category name
+    category_id_to_category_name = {}
+    for category in categories:
+        category_id_to_category_name[category['id']] = category['name']
+
+    # create a list of pairs of (filename, metadata)
+    custom_metadata = []
+    for image in images:
+
+        # we want to count the number of instances of every category on the image
+        metadata = {'number_of_instances': {}}
+        for category in categories:
+            metadata['number_of_instances'][category['name']] = 0
+
+        # count all annotations for that image
+        for annotation in annotations:
+            if annotation['image_id'] == image['id']:
+                metadata['number_of_instances'][category_id_to_category_name[annotation['category_id']]] += 1
+
+        # append (filename, metadata) to the list
+        custom_metadata.append((image['file_name'], metadata))
+
+    # save custom metadata in the correct json format
+    save_custom_metadata(OUTPUT_FILE, custom_metadata)
+
+
+Now that we have extracted and saved the custom metadata in the correct format,
+we can upload the dataset to the `Lightly web-app <https://app.lightly.ai>`_. 
+Don't forget to replace the variables `YOUR_TOKEN`, and `YOUR_CUSTOM_METADATA_FILE.json` in the command below.
+
+.. code-block:: bash
+
+    lightly-magic input_dir="./aquarium" trainer.max_epochs=0 token=YOUR_TOKEN new_dataset_name="Aquarium" custom_metadata=YOUR_CUSTOM_METADATA_FILE.json
+
+
+Note that if you already have a dataset on the Lightly platform, you can add custom metadata with:
+
+.. code-block:: bash
+
+    lightly-upload token=YOUR_TOKEN dataset_id=YOUR_DATASET_ID custom_metadata=YOUR_CUSTOM_METADATA_FILE.json
+
+
+.. note::
+
+    Check out :py:class:`lightly.api.api_workflow_client.ApiWorkflowClient.upload_custom_metadata`
+    to learn more about the expected format of the custom metadata json file.
+
+
+Configuration
+---------------
+In order to use custom metadata in the web-app, it needs to be configured first. For this, head to your dataset and 
+select the `Settings > Custom Metadata` menu. We name the configuration `Number of Fish` since we're interested in the
+number of fish on each image. Next, we fill in the `Path`: To do so, we click on it and select `number_of_instances.fish`.
+Finally, we pick `Integer` as a data type. The configured custom metadata should look like this:
+
+.. figure:: ../../tutorials_source/platform/images/custom_metadata_configuration_aquarium.png
+   :align: center
+   :alt: Custom metadata configuration for number of fish
+
+   Completed configuration for the custom metadata "Number of Fish".
+
+
+To verify the configuration works correctly, we can head to the `Explore` screen and sort our dataset by the `Number of Fish`.
+If we decrease in descending order we can easily verify that the metadata is configured properly.
```

Comment:
What is meant by "decrease in descending order"? Do you mean "If we sort in descending order, we see that images many fish are shown first. This verifies that the metadata is configured properly.